현정빈/RestApiStudy/src/main/java/com/example/restapistudy/domain/Movie.java

```diff
@@ -0,0 +1,30 @@
+package com.example.restapistudy.domain;
+
+import jdk.jshell.Snippet;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class Movie
+{
+    private Long id; //아이디
+    private String title; //영화제목
+    private String director; //감독
+    private int releaseYear; //출시연도
+    private double rating; //평점
+
+    @Builder
+    public Movie(Long id, String title, String director, int releaseYear, double rating)
+    {
+        this.id = id;
+        this.title =title;
+        this.director = director;
+        this.releaseYear = releaseYear;
+        this.rating = rating;
+    }
+
+    public void initId(Long id)
+    {this.id = id;}
```

Comment:
개행!

Reply:
모든 클래스, 메서드에는 아래 주어진 코드와 같이 한 줄에 중괄호 시작점을 작성하시는게 관례이니 다음부터는 꼭 이렇게 해봐요!
```suggestion
    public void initId(Long id) {
        this.id = id;
    }
```

Reply:
앗 네 다음 과제부터 꼭 지키도록 하겠습니다!